packages/editor/src/components/header/index.js

```diff
@@ -142,9 +146,13 @@ function Header( {
 					forceIsAutosaveable={ forceIsDirty }
 				/>
 				<PostViewLink />
+
+				{ zoomOutExperimentEnabled && <ZoomOutToggle /> }
+
 				{ ( isWideViewport || ! showIconLabels ) && (
 					<PinnedItems.Slot scope="core" />
```

Comment:
I noticed we have this new slot for pinned items. I don't believe we should use this slot to add ZoomOutToggle as it will be a key part of the UI.